requirements.txt

```diff
@@ -207,16 +206,16 @@ sqlalchemy-utils==0.34.2
 sqlalchemy==1.3.20; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
 sqlparse==0.4.1; python_version >= '3.5'
 starlette==0.13.6; python_version >= '3.6'
-subprocess32==3.5.4; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2' and python_version < '4'
+subprocess32==3.5.4; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3' and python_version < '4'
 sympy==1.6.2; python_version >= '3.5'
 tabulate==0.8.7
 terminado==0.9.1; python_version >= '3.6'
 testpath==0.4.4
 threadpoolctl==2.1.0; python_version >= '3.5'
-toml==0.10.2; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2'
-topika==0.2.1
+toml==0.10.2; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'
+topika==0.2.2
 tornado==4.5.3
-tqdm==4.51.0; python_version >= '2.6' and python_version not in '3.0, 3.1'
+tqdm==4.51.0; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'
```

Comment:
Do you know why pipenv constantly changes its mind on the excluded Python versions?

Reply:
I've absolutely no idea... The Pipenv version I am using is the same.

Reply:
Odd.